frontend/src/features/public-form/components/FormPaymentPage/stripe/components/DownloadReceiptBlock.tsx

```diff
@@ -34,22 +39,59 @@ const PaymentSummaryRow = ({
       direction={{ base: 'column', md: 'row' }}
       spacing={{ base: 0, md: '1.5rem' }}
     >
-      <Text textStyle="body-2" width="6.5rem" color="content.medium">
+      <Text textStyle="body-1" width="6.5rem" color="secondary.400">
+        {title}
+      </Text>
+      <Text textStyle="body-1" color="secondary.700">
+        {input}
+      </Text>
+    </Stack>
+  )
+}
+
+const PaymentSummaryRow = ({
+  title,
+  input,
+}: {
+  title: string
+  input: string
+}): JSX.Element => {
+  return (
+    <Stack
+      direction={{ base: 'row', md: 'row' }}
+      spacing={{ base: 0, md: '3rem' }}
+      justify={{ base: 'space-between', md: 'space-between' }}
```

Comment:
I see that `base` and `md` are using the same `row` / `space-between`, if we replace them with `direction="row"`, and `justify="space-between"`, would it work?

Reply:
yes it works!! 🙏🏼